tests/datasource/data_connector/test_inferred_asset_dbfs_data_connector.py

```diff
@@ -5,6 +5,7 @@
 import boto3
 import botocore
 import pytest
+from pyfakefs.fake_filesystem import FakeFilesystem
```

Comment:
This should be a `if TYPE_CHECKING:` import

Reply:
@Kilo59 Done.  Thanks.